threaded/_base_gthreadpooled.py

```diff
@@ -18,6 +18,7 @@
 
 import typing  # noqa  # pylint: disable=unused-import
 
+import gevent.event  # noqa  # pylint: disable=unused-import
```

Comment:
why?

Reply:
It's required for valid typecheck, but not used for the main functionality.

Reply:
Used on L94

Reply:
Ah yep, maybe next time better a dedicate commit for typing annotation?